### ISSUE TITLE
Missing text for 'user login' default label (all languages)

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#21917]
### Steps to reproduce the issue
Go to user login

### Expected result
Field caption should be 'User Login'

### Actual result
Field caption is placeholder "COM_USERS_LOGIN_DEFAULT_LABEL 

### System information (as much as possible)
still missing in Joomla 3.8.12

### Additional comments
=== Affects all languages ===
 file:       /language/xx-XX/xx-XX.com_users.ini
 line:       any
 is:         none
 should be:  COM_USERS_LOGIN_DEFAULT_LABEL="User Login"
 Temporarily put in 'overrides' folder as a workaround.

**Comment by ghost** [comment]
You talk about Frontend? Can you give Details (Screenshot) where to look?

**Comment by brianteeman** [comment]
Can you please post a screenshot of where you are seeing this?

**Comment by richard67** [comment]
I can confirm that the language string is used in the xml but defined nowehere, and there seems also not to be another language string for that purpose:

```
F:\Users\Richard\Documents\GitHub\joomla-cms>findstr /s /c:"COM_USERS_LOGIN" *.*
components\com_users\models\forms\login.xml:    <fieldset name="credentials" label="COM_USERS_LOGIN_DEFAULT_LABEL">
components\com_users\models\forms\login.xml:                    label="COM_USERS_LOGIN_USERNAME_LABEL"
components\com_users\views\login\tmpl\default_login.php:                <img src="<?php echo $this->escape($this->params->get('login_image')); ?>" class="login-image" alt="<?php echo JText::_('COM_USERS_LOGIN_IMAGE_ALT'); ?>" />
components\com_users\views\login\tmpl\default_login.php:                                                        <?php echo JText::_('COM_USERS_LOGIN_REMEMBER_ME'); ?>
components\com_users\views\login\tmpl\default_login.php:                                <?php echo JText::_('COM_USERS_LOGIN_RESET'); ?>
components\com_users\views\login\tmpl\default_login.php:                                <?php echo JText::_('COM_USERS_LOGIN_REMIND'); ?>
components\com_users\views\login\tmpl\default_login.php:                                        <?php echo JText::_('COM_USERS_LOGIN_REGISTER'); ?>
language\en-GB\en-GB.com_users.ini:COM_USERS_LOGIN_IMAGE_ALT="Login image"
language\en-GB\en-GB.com_users.ini:COM_USERS_LOGIN_REGISTER="Don't have an account?"
language\en-GB\en-GB.com_users.ini:COM_USERS_LOGIN_REMEMBER_ME="Remember me"
language\en-GB\en-GB.com_users.ini:COM_USERS_LOGIN_REMIND="Forgot your username?"
language\en-GB\en-GB.com_users.ini:COM_USERS_LOGIN_RESET="Forgot your password?"
language\en-GB\en-GB.com_users.ini:COM_USERS_LOGIN_USERNAME_LABEL="Username"

F:\Users\Richard\Documents\GitHub\joomla-cms>findstr /s /c:"COM_USERS_LOGIN_DEFAULT_LABEL" *.*
components\com_users\models\forms\login.xml:    <fieldset name="credentials" label="COM_USERS_LOGIN_DEFAULT_LABEL">

F:\Users\Richard\Documents\GitHub\joomla-cms>
```

**Comment by brianteeman** [comment]
@richard67 that's nothing to do with the report though is it

**Comment by richard67** [comment]
@brianteeman Well it shows that beside the components\com_users\models\forms\login.xml, the core does not use this language string anywhere, so it could be a 3rd party extension using it in any php file. But the fact that the core uses it in the xml also shows that it should be defined in the language file.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21917">issues.joomla.org/tracker/joomla-cms/21917</a>.</sub>

**Comment by ghost** [comment]
**@brianteeman** yes of course it has !
I can make a screenshot but it will be in German. My additional comments show exactly where to look at. 

**@franz-wohlkoenig** Sorry, I forgot to mention that I talk about 'frontend' .

**@richard67** Thanks for your confirmation.

Thanks to you all.

**Comment by richard67** [comment]
Pull request is PR #21919 . Please text and review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21917">issues.joomla.org/tracker/joomla-cms/21917</a>.</sub>

**Comment by brianteeman** [comment]
> But the fact that the core uses it in the xml also shows that it should be defined in the language file.

Or it needs to be removed from the xml if it isnt used

**Comment by brianteeman** [comment]
@madmetz please post a screenshot because as far as i can tell the core of Joomla never uses that string

**Comment by ghost** [comment]
**@brianteenman** OK, here is a part of my login screen. I marked the filed with a yellow circle.
![my_login_screen](https://user-images.githubusercontent.com/42808721/44850315-1475ca80-ac5d-11e8-9e73-f59c29fe5154.jpg)
The word "Benutzeranmeldung" (English: 'User Login') comes from this string / placeholder.

**Comment by ghost** [comment]
@madmetz can you test using a Core-Template (Protostar)? And where to see "Field caption is placeholder "COM_USERS_LOGIN_DEFAULT_LABEL"?

**Comment by mbabker** [comment]
The core layout doesn't render this fieldset's label.  The template on this website is.  You're not going to reproduce this with core.

**Comment by richard67** [comment]
@mbabker So what do you recommend? Replace the language constant in the components\com_users\models\forms\login.xml file by a hard-coded string "User Login"?

**Comment by mbabker** [comment]
No.  Your PR is just fine.

**Comment by ghost** [comment]
closed as having Pull Request #21919
